easybuild/easyconfigs/b/bigdft-suite/bigdft_suite_index_bug.patch

```diff
@@ -0,0 +1,21 @@
+# same effect as patch for futile (futile_index_bug.patch), except due to directory structure, must be regenerated
```

Comment:
I would also mention `fix the redefinition of an index variable caught by GCC 11`, to make it more descriptive (and then fix the SHA256 checksum for the patch accordingly)

Reply:
done